hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSClientRetries.java

```diff
@@ -406,14 +407,14 @@ public void testLeaseRenewSocketTimeout() throws Exception
       // throws SocketTimeoutException.
       Mockito.doNothing().when(spyNN).renewLease(
           Mockito.anyString());
-      leaseRenewer = client.getLeaseRenewer();
-      leaseRenewer.setRenewalTime(100);
+      LeaseRenewer leaseRenewer2 = client.getLeaseRenewer();
```

Comment:
Should we set final to leaseRenewer2 as well as leaseRenewer1?

Reply:
I'm +1 if that is addressed.

Reply:
I addressed your comment. Thanks.